web-ui/src/main/resources/catalog/js/CatController.js

```diff
@@ -421,11 +425,12 @@
 
 
       //Update Links for social media
+      $scope.showSocialBarInFooter = gnGlobalSettings.gnCfg.mods.footer.showSocialBar;
```

Comment:
This does not need to go on the scope as it is only used in the `locationChangeSuccess` listener.

Reply:
True, I will change it